draft-packages/form/KaizenDraft/Form/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+
+import { InputSearch } from "../Primitives/InputSearch"
+import { FieldGroup, Label } from ".."
+export interface SearchFieldProps
+  extends Omit<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    "className" | "defaultValue"
+  > {
+  reversed?: boolean
+  loading?: boolean
+  secondary?: boolean
+  labelText?: React.ReactNode
```

Comment:
Sorry just picked this up - this is usually required and I think we still want it required here. In the case that's it's `secondary` variant we'll want to put it as an `aria-label` on the input.

Reply:
Using it for an `aria-label` will also require this to be a string rather than a `ReactNode` as well.